descriptors/analysis_engines/demo.xml

```diff
@@ -141,14 +141,14 @@
     <flowConstraints>
       <fixedFlow>
        <node>CollectionReader</node>
-       <node>ImagePreprocessor</node>
+      <!-- <node>ImagePreprocessor</node>
        <node>PointCloudFilter</node>
        <node>NormalEstimator</node>
        <node>PlaneAnnotator</node>
        <node>ImageSegmentationAnnotator</node>
        <node>PointCloudClusterExtractor</node>
        <node>ClusterMerger</node>
-       <node>ResultAdvertiser</node>
+       <node>ResultAdvertiser</node>-->
       </fixedFlow>
     </flowConstraints>
     <typePriorities/>
```

Comment:
Changes to demo.xml should not be committed. Will break the tutorial.

Reply:
yea, you're right:) I'll uncomment;